src/library/trackset/crate/cratetablemodel.cpp

```diff
@@ -124,6 +124,7 @@ TrackModel::CapabilitiesFlags CrateTableModel::getCapabilities() const {
             qWarning() << "Failed to read create" << m_selectedCrate;
         }
     }
+    
```

Comment:
trailing whitespace

Reply:
I think that was intentional to separate the different parts of the function

Reply:
A blank line is okay, but not trailing whitespace (e.g. spaces)

Reply:
Ah, didn't see that in the diff, will fix.